Makefile

```diff
@@ -69,77 +96,126 @@ vet:
+	export CONTAINER_CMD=$(CONTAINER_CMD); \
+	bazel run operator:forklift-operator-bundle-image \
+		--verbose_failures \
+		--action_env CONTAINER_CMD=$(CONTAINER_CMD) \
+		--action_env VERSION=$(VERSION) \
+		--action_env NAMESPACE=$(NAMESPACE) \
+		--action_env CHANNELS=$(CHANNELS) \
+		--action_env DEFAULT_CHANNEL=$(DEFAULT_CHANNEL) \
+		--action_env OPERATOR_IMAGE=$(OPERATOR_IMAGE) \
+		--action_env MUST_GATHER_IMAGE=$(MUST_GATHER_IMAGE) \
+		--action_env MUST_GATHER_API_IMAGE=$(MUST_GATHER_API_IMAGE) \
+		--action_env UI_IMAGE=$(UI_IMAGE) \
+		--action_env UI_PLUGIN_IMAGE=$(UI_PLUGIN_IMAGE) \
+		--action_env VALIDATION_IMAGE=$(VALIDATION_IMAGE) \
+		--action_env VIRT_V2V_IMAGE=$(VIRT_V2V_IMAGE_COLD) \
+		--action_env CONTROLLER_IMAGE=$(CONTROLLER_IMAGE) \
+		--action_env API_IMAGE=$(API_IMAGE) \
 
 push-operator-bundle-image: build-operator-bundle-image
-	 $(CONTAINER_CMD) tag operator:forklift-operator-bundle-image ${OPERATOR_BUNDLE_IMAGE}
-	 $(CONTAINER_CMD) push ${OPERATOR_BUNDLE_IMAGE}
-
-build-operator-index-image:
-	bazel run operator:forklift-operator-index-image --action_env REGISTRY=${REGISTRY} --action_env REGISTRY_ACCOUNT=${REGISTRY_ACCOUNT} --action_env REGISTRY_TAG=${REGISTRY_TAG}
+	 $(CONTAINER_CMD) tag bazel/operator:forklift-operator-bundle-image $(OPERATOR_BUNDLE_IMAGE)
+	 $(CONTAINER_CMD) push $(OPERATOR_BUNDLE_IMAGE)
+
+build-operator-index-image: check_container_runtmime
+	export CONTAINER_CMD=$(CONTAINER_CMD); \
+	bazel run operator:forklift-operator-index-image \
+		--verbose_failures \
+		--sandbox_writable_path=$${XDG_RUNTIME_DIR} \
+		--action_env CONTAINER_CMD=$(CONTAINER_CMD) \
+		--action_env VERSION=$(VERSION) \
+		--action_env CHANNELS=$(CHANNELS) \
+		--action_env DEFAULT_CHANNEL=$(DEFAULT_CHANNEL) \
+		--action_env OPT_OPTS=$(OPM_OPTS) \
+		--action_env REGISTRY=$(REGISTRY) \
+		--action_env REGISTRY_TAG=$(REGISTRY_TAG) \
+		--action_env REGISTRY_ACCOUNT=$(REGISTRY_ACCOUNT)
 
 push-operator-index-image: build-operator-index-image
-	$(CONTAINER_CMD) tag operator:forklift-operator-index-image ${OPERATOR_INDEX_IMAGE}
-	$(CONTAINER_CMD) push ${OPERATOR_INDEX_IMAGE}
+	$(CONTAINER_CMD) tag bazel/operator:forklift-operator-index-image $(OPERATOR_INDEX_IMAGE)
+	$(CONTAINER_CMD) push $(OPERATOR_INDEX_IMAGE)
+
+build-all-images: build-api-image build-controller-image build-validation-image build-operator-image build-virt-v2v-image build-virt-v2v-warm-image build-operator-bundle-image build-operator-index-image
 
 push-all-images: push-api-image push-controller-image push-validation-image push-operator-image push-virt-v2v-image push-virt-v2v-warm-image push-operator-bundle-image push-operator-index-image
 
+.PHONY: check_container_runtmime
+check_container_runtmime:
+	  @if [ ! -x "$(CONTAINER_CMD)" ]; then \
+        echo "Container runtime was not automatically detected"; \
+        echo "Please install podman or docker and make sure it's available in PATH"; \
+        exit 1; \
+    fi
```

Comment:
the indentation here seems wrong, @mmartinv can you please check why?

Reply:
Done